interpreter/cling/lib/Interpreter/ClingPragmas.cpp

```diff
@@ -60,6 +78,24 @@ namespace {
       Token& CurTok = const_cast<Token&>(P.getCurToken());
       CurTok.setKind(tok::semi);
 
+      return Literal;
```

Comment:
Enforce that `Literal` is non-empty; `llvm::errs() <<...` otherwise and return `false`.